packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -0,0 +1,359 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import DatePicker from 'terra-date-picker';
+import TimeInput from 'terra-time-input';
+import DateUtil from 'terra-date-picker/lib/DateUtil';
+import styles from './DateTimePicker.scss';
+import DateTimeUtil from './DateTimeUtil';
+import TimeClarification from './_TimeClarification';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * An array of ISO 8601 string representation of the dates to disable in the picker.
+   */
+  excludeDates: PropTypes.arrayOf(PropTypes.string),
+  /**
+   * A function that gets called for each date in the picker to evaluate which date should be disabled. A return value of true will be enabled and false will be disabled.
+   */
+  filterDate: PropTypes.func,
+  /**
+   * An array of ISO 8601 string representation of the dates to enable in the picker. All Other dates will be disabled.
+   */
+  includeDates: PropTypes.arrayOf(PropTypes.string),
+  /**
+   * Custom input attributes to apply to the date and time inputs. Use the name prop to set the name for the input. Do not set the name in inputAttribute as it will be ignored.
+   */
+  inputAttributes: PropTypes.object,
```

Comment:
I'm using the same InputAttributes prop for both the date input and time input because I didn't think that there would be a use case where we would want to apply one set of attributes for the date input and a different set for the time input. If there is such use case, I can certainly separate this prop.